components/dashboard/src/teams/JoinTeam.tsx

```diff
@@ -27,13 +29,17 @@ export default function JoinTeamPage() {
                 const result = await organizationClient.joinOrganization({ invitationId: inviteId });
                 orgInvalidator();
 
-                history.push(`/members?org=${result.organizationId}`);
+                if (!isDataOps) {
```

Comment:
This is not dataops dependent but whether the user is a collaborator. I don't think redirecting new joiners to the mebers list is a good idea anyway. so let's redirect everyone to the main page.